testing/vcs/test_vcs_mercator_edge.py

```diff
@@ -0,0 +1,21 @@
+import vcs,cdms2
+import os,sys
+f = cdms2.open(os.path.join(vcs.sample_data, "clt.nc"))
+s = f("clt")
+x = vcs.init()
+iso = x.createisofill()
+iso.projection = "mercator"
+x.plot(s(latitude=(-90, 90)), iso)
```

Comment:
I suspect the slight diff on my ubunutu maybe due to default size being to small.